src/classes/PDF.ts

```diff
@@ -58,6 +96,60 @@ export default class PDF extends OneBlinkAPI {
     return response.buffer()
   }
 
+  
+  /**
+   * #### Example
+   * ```javascript
+   * const fs = require('fs')
+   * const util = require('util')
+   * 
+   * const writeFileAsync = util.promisify(fs.writeFile)
+   * 
+   * async function run() {
+   *   const buffer = await pdf.generatePDF({
+   *     body: {
+   *       html: `
+   *         <p>I will be in the middle</p>
+   *       `,
+   *     },
+   *     header: {
+   *       html: `
+   *       <div style="font-size: 9px; margin: 0 15px; width: 100%; text-align: center;">
+   *         I will be at the top of every page
+   *       </div>
+   *       `,
+   *     },
+   *     footer: {
+   *       html: `
+   *       <div style="font-size: 9px; margin: 0 15px; width: 100%; text-align: center;">
+   *         I will be at the bottom of every page ({_BLINK_PAGE_NO_}/{_BLINK_PAGES_})
+   *       </div>
+   *       `,
+   *     },
+   *     page: {
+   *       orientation: 'Portrait',
+   *       size: 'A4',
+   *       margins: {
+   *         top: '15mm',
+   *         right: '5mm',
+   *         bottom: '15mm',
+   *         left: '5mm',
+   *       },
+   *     },
+   *   })
+   *   await writeFileAsync('./custom.pdf', buffer, 'binary')
+   * }
+   * 
+   * @param options.body.html The HTML to render as a pdf
+   * @param options.header.html The HTML to render at the top of every page
+   * @param options.footer.html The HTML to render at the bottom of every page
+   * @param options.page.orientation `'Portrait'` or `'Landscape`'. Default is `'Portrait'`
+   * @param options.page.size `'Letter'`, `'Legal'`, `'Tabloid'`, `'Ledger'`, `'A0'`, `'A1'`, `'A2'`, `'A3'`, `'A4'`, `'A5'` or `'A6'`. Default is `'A4'`
+   * @param options.page.margins.top How much space between the top of each page and the content. Supported dimension units are: `'mm'`, `'cm'`, `'in'`, `'px'` 
+   * @param options.page.margins.bottom How much space between the bottom of the page and content
+   * @param options.page.margins.right How much space between the right of each page and the content. Supported dimension units are: `'mm'`, `'cm'`, `'in'`, `'px'`
+   * @param options.page.margins.left How much space between the left of each page and the content. Supported dimension units are: `'mm'`, `'cm'`, `'in'`, `'px'`
+   * ``` */
```

Comment:
These `@param` tags can be removed, since the type is something we are exporting from types, meaning users will be able to click through to it in the docs, at which point they should see the descriptions. What this means is that these descriptions will need to live on the actual type, which is in our `@oneblink/types` repo: https://github.com/oneblink/types.

However, we are doing this at a later date, so no need for you to do it now, just remove these ones.